python/nano/test/pytorch/tests/test_inference_pipeline_ipex.py

```diff
@@ -356,3 +356,20 @@ def metric(model, data_loader):
                                metric=metric,
                                direction="max",
                                thread_num=4)
+
+    def test_multi_instance(self):
+        model = Net()
+        model.eval()
+        inference_opt = InferenceOptimizer()
+        multi_instance_model = inference_opt.to_multi_instance(model, num_processes=2)
+
+        test_loader = create_data_loader(self.data_dir, 1, self.num_workers, data_transform, subset=50, shuffle=False)
+        input_data = list(map(lambda b: b[0], test_loader))
```

Comment:
No, `multi_instance_model` and `model` are the same expect `multi_instance_model` receives a list and return a list.

It can work well as long as the input of `multi_instance_model` is a list of batches.

Here I used all data only for convenience.